project-resources/REQUESTING_GITHUB_RESOURCES.md

```diff
@@ -30,6 +30,9 @@ The process for requesting Github resources is:
 
 The Pull Request template below should be completed with a Pull Request containing an addition to the `GITHUB_REQUESTS/{CurrentYear}.md` spreadsheet from this repository, filling the required information.
 
+> [!NOTE]\
+> The costs are are billed on a monthly-basis. Hence the the costs should be calculated on a monthly-basis.
```

Comment:
```suggestion
> The costs are billed on a monthly-basis. Hence the costs should be calculated on a monthly-basis.
```

Reply:
I don't think "monthly basis" should be hyphenated.

Reply:
@ljharb what should the change be here then?

Reply:
```suggestion
> The costs are billed on a monthly basis. Hence the costs should be calculated on a monthly basis.
```